action-run-cmd-create-pr/README.md

```diff
@@ -0,0 +1,24 @@
+# action-run-cmd-create-pr
+
+A GitHub action for running a command to change a remote repository, make a commit and create a PR
+
+## Available tools
+
+[see DockerFile](/action-run-cmd-create-pr/Dockerfile)
+
+## Example workflow
+
+```yaml
+- name: Update version
+  uses: qlik-oss/ci-tools/action-run-cmd-create-pr@master
+  with:
+      gh_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      owner: qlik-oss
+      repository: ci-tools
+      branch: update_version_to_${{ env.LATEST_VERSION }}
+      command: "command_to_run"
```

Comment:
For clarity, I think

"command_to_run"

should be changed to

<some_path>/version_updater.sh